### DISCUSSION TITLE
Ways to quantify gene expression activity

### DISCUSSION THREAD

**Discussion by sr320** [sr320/ceabigr#2] [Ideas]
I think it is important to first think about all the ways one might quantify gene expression activity.. 

Here is a start...

- gene level expression (fpkm)
- transcript level expresssion (fpkm)
- number of transcripts per gene expressed
- co-efficient of variation of gene expression across treatment
- co-efficient of variation of transcript expression within gene

other ideas?

**Comment by laurahspencer** [comment]
If there are any plans to use DESeq2 to explore differential expression, we'll also need raw gene counts (not normalized) - see http://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#why-un-normalized-counts

(Which may be covered by your 3rd bullet, above?)
